README.md

```diff
@@ -1,39 +1,35 @@
-# Sentry On-Premise
+# Sentry On-Premise [![Build Status][build-status-image]][build-status-url]
 
 Official bootstrap for running your own [Sentry](https://sentry.io/) with [Docker](https://www.docker.com/).
 
 ## Requirements
 
  * Docker 1.10.0+
- * Compose 1.17.0+ _(optional)_
- 
- ## Minimum Hardware Requirements:
- 
- * You need at least 3GB Ram
-
-## Up and Running
-
-Assuming you've just cloned this repository, the following steps
-will get you up and running in no time!
-
-There may need to be modifications to the included `docker-compose.yml` file to accommodate your needs or your environment. These instructions are a guideline for what you should generally do.
-
-1. `docker volume create --name=sentry-data && docker volume create --name=sentry-postgres` - Make our local database and sentry volumes
-    Docker volumes have to be created manually, as they are declared as external to be more durable.
-2. `cp -n .env.example .env` - create env config file
-3. `docker-compose build` - Build and tag the Docker services
-4. `docker-compose run --rm web config generate-secret-key` - Generate a secret key.
-    Add it to `.env` as `SENTRY_SECRET_KEY`.
-5. `docker-compose run --rm web upgrade` - Build the database.
-    Use the interactive prompts to create a user account.
-6. `docker-compose up -d` - Lift all services (detached/background mode).
-7. Access your instance at `localhost:9000`!
+ * Compose 1.17.0+
```

Comment:
I’d still leave the note that this is optional. Running Sentry does not have some hidden dependency on Docker Compose, and what’s provided is still an example. Ideally users use whatever scheduler they want.

Reply:
I'd say when we have an "official" repo, it is better to reduce the surface area we cover. What we guarantee here in the repo is the flow with `docker-compose` which should be okay for local or small deploys. I'm sure folks who want to use something fancier wouldn't be bothered by this part :)